version/version.go

```diff
@@ -1,5 +1,5 @@
 package version
 
 var (
-	Version = "1.1.0"
+	Version = "1.2.0"
```

Comment:
This fails some tests in pkg/controller/hyperconverged/hyperconverged_controller_test.go

To fix the tests, it is possible to replace https://github.com/tiraboschi/hyperconverged-cluster-operator/blob/b25a7dfb9dc399c369b7a9dd1e78be0df546916d/pkg/controller/hyperconverged/hyperconverged_controller_test.go#L1963 with this code:
``` golang
BeforeEach(func() {
	os.Setenv(util.HcoKvIoVersionName, version.Version)
})
```